server/odc-common/src/main/java/com/oceanbase/odc/common/concurrent/Await.java

```diff
@@ -103,7 +105,7 @@ private ScheduledFuture<?> getScheduledFuture(AtomicInteger hasRetryTimes, Count
                         countDownLatch.countDown();
                     }
                 },
-                0, period, timeUnit);
+                0, period, periodTimeUnit);
```

Comment:
what's the different between `periodTimeUnit` and `timeUnit`

Reply:
periodTimeUnit is represent check condition period, and timeUnit represent await timeoutUnit, but I refactor old variable `timeUnit` to `timeoutUnit`